source/includes/_endpoint-multitable.md

```diff
@@ -0,0 +1,88 @@
+## Multitables
+
+### Catalog
+
+`/datasets/{dataset_id}/multitables/`
+
+#### GET
+
+```json
+{
+    "element": "shoji:catalog",
+    "self": "/api/datasets/123/multitables/",
+    "specification": "/api/specifications/multitables/",
+    "description": "List of multitable definitions for this dataset",
+    "index": {
+        "/api/datasets/123/multitables/7ab1e/": {
+            "is_public": false,
+            "owner_id": "/api/users/b055/",
+            "name": "Basic Demographics",
+            "id": "7ab1e"
+        }
+    }
+}
+```
+
+GET on this resource returns a Shoji Catalog with the list of Multitables that the
+current user can use on this Dataset.
+
+This index contains two kinds of multitables: those that belong to the dataset, denoted by the `is_public` tuple attribute; and those that belong to the current user. Personal multitables are those created by the authenticated user, and they cannot be accessed by other users. Dataset multitables are available to all users who are authorized to view the dataset.
+
+#### POST
+
+POST a Shoji Entity to this catalog to create a new multitable definition. Entities must include a `name` and `template`; the template must contain a series of objects with keys `variable` and `query` and optionally `transform`. If omitted, `is_public` defaults to `false`. A successful POST yields a 201 response
```

Comment:
This section should also link appropriately to the feature guide that talks about multitables; that feature guide should link back to here as appropriate and delete the duplicated content from there.